src/components/common/icons/Progressing.tsx

```diff
@@ -5,12 +5,13 @@ interface ProgressingProps {
     loadingText?: string;
     size?: number;
     fullHeight?: boolean;
+    theme?: 'white' | 'default';
 }
 
-export function Progressing({ pageLoader, size }: ProgressingProps): JSX.Element {
+export function Progressing({ pageLoader, size, theme }: ProgressingProps): JSX.Element {
     const loaderSize = size ? `${size}px` : pageLoader ? '48px' : '20px';
     return (
-        <div className="loader">
+        <div className={`loader ${theme || 'default'}-background`}>
```

Comment:
Is `default-background`classname adding any value here?

Reply:
It's just a placeholder to stop including `undefined-background` as a class name :)